commands/create_cluster_test.go

```diff
@@ -26,37 +27,43 @@ func TestReadFiles(t *testing.T) {
 
 // Test_CreateFromYAML01 tests parsing a most simplistic YAML definition.
 func Test_CreateFromYAML01(t *testing.T) {
-	definition := clusterDefinition{}
+	def := clusterDefinition{}
 	data := []byte(`owner: myorg`)
-	myDef, err := unmarshalDefinition(data, definition)
```

Comment:
I removed `unmarshalDefinition` as it was a rather useless indirect to the actual YAML library.